panther_analysis_tool/lookup_table.py

```diff
@@ -0,0 +1,174 @@
+"""
```

Comment:
I could but my team doesn't "own" this code + it's public + even if we did own this code, I'm really not trying to overhaul everything because my first thought is the several other files in this directory should also be their own packages.

Reply:
OK, I'll do it. :)